core/types/receipt.go

```diff
@@ -397,6 +409,14 @@ func (rs Receipts) EncodeIndex(i int, w *bytes.Buffer) {
 	}
 }
 
+func L1Cost(rollupDataGas uint64, l1BaseFee, overhead, scalar *big.Int) *big.Int {
+	l1GasUsed := new(big.Int).SetUint64(rollupDataGas)
+	l1GasUsed = l1GasUsed.Add(l1GasUsed, overhead) // overhead = 2100
```

Comment:
the overhead isn't a constant and will likely change post bedrock so i'm not sure if having that comment is useful

Reply:
Will fix